### ISSUE TITLE
Adding more test

### ISSUE THREAD

**Issue by RemiDesgrange** [sid5432/pyOTDR#6]
I'm working on makeing python3 fully support, and found that test pass and running pyOTDR against `data/` provided sor file fail. So I'm suggesting adding more test.

I'm planning to do this when #5 will be merged

**Comment by sid5432** [comment]
Hi Rémi: I have merged your changes! There were a few minor tweaks (some only cosmetic), but everything seems to run correctly, and the regression tests all pass. Thanks!

There was one line in supparams.py (line 28) that initially caused an error. It turns out there was an extra white-space character: in the line '{} {} block starting position unknown', the space between '}' and '}' was actually two bytes 0xC2 and 0xA0 (instead of 0x20). It probably has to do with my computer's locale and character encoding setting.

The other place that confused me for a while was line 111 in datapts.py: the indentation was wrong for some reason.

I've added a line in main.py so that the debug messages are printed.
